be/src/exec/analytor.cpp

```diff
@@ -462,7 +465,8 @@ void Analytor::create_agg_result_columns(int64_t chunk_size) {
             // binary column cound't call resize method like Numeric Column,
             // so we only reserve it.
             if (_agg_fn_types[i].result_type.type == LogicalType::TYPE_CHAR ||
-                _agg_fn_types[i].result_type.type == LogicalType::TYPE_VARCHAR) {
+                _agg_fn_types[i].result_type.type == LogicalType::TYPE_VARCHAR ||
+                _agg_fn_types[i].result_type.type == LogicalType::TYPE_ARRAY) {
```

Comment:
what about JSON,MAP,STRUCT, and other special types?

Reply:
Currently, we don't support those types you mentioned as result type. But it seems ok to be included.